tests/unit/output/test_local.py

```diff
@@ -20,6 +21,29 @@ def test_save_missing(self):
             with self.assertRaises(o.DoesNotExistError):
                 o.save()
 
+    def test_str_if_inside_repo(self):
+        o = self._get_output()
+        self.assertEqual("path", str(o))
+
+    def test_str_if_workdir_is_inside_repo(self):
+        stage = Stage(self.dvc)
+        work_dir = os.path.join(stage.repo.root_dir, "models")
+        os.makedirs(work_dir, exist_ok=False)
+        os.chdir(work_dir)
+
+        o = self._get_cls()(stage, "v1/model.pkl", cache=False)
+        self.assertEqual("v1/model.pkl", str(o))
+
+        o = self._get_cls()(stage, "../data/dataset-interim.csv", cache=False)
+        self.assertEqual("../data/dataset-interim.csv", str(o))
```

Comment:
- [ ] Fix the test.

Reply:
I'll mark it "Ready For Review" after fixing this test. I'm not quite sure when the `chdir` occurs.